pkg/agent/openflow/pipeline.go

```diff
@@ -1604,17 +1646,35 @@ func (f *featureNetworkPolicy) allowRulesMetricFlows(conjunctionID uint32, ingre
 	// The flow matching 'ct_state=+new' tracks the number of sessions and byte count of the first packet for each
 	// session.
 	// The flow matching 'ct_state=-new' tracks the byte/packet count of an established connection (both directions).
```

Comment:
These comments should be moved to line 1659?

Reply:
done